core-bundle/src/Resources/config/listener.yml

```diff
@@ -140,6 +140,17 @@ services:
             # The priority must be higher than the one of the Twig exception listener (defaults to -128)
             - { name: kernel.event_listener, event: kernel.exception, method: onKernelException, priority: -96 }
 
+    contao.listener.preview_authentication:
+        class: Contao\ManagerBundle\EventListener\PreviewAuthenticationListener
```

Comment:
The class name still references `ManagerBundle`. Is this correct?

Reply:
Copy pasta 🙈 Fixed in 2e4011d15.